src/redux/books/book.js

```diff
@@ -0,0 +1,26 @@
+// REDUCER
+const reducer = (state = [], action) => {
+  switch (action.type) {
+    default:
+      return state;
+    case 'add':
+      return state;
+    case 'remove':
+      return state;
```

Comment:
- [ ] Please edit the implementation of the "add" and "remove" reducers so that the "add" would return the state and concatenate the new book on it, while the remove would remove the book using the action payload.

- [ ] Kindly edit the switch statement so that the default case would be at the end as a best practice.
```
 switch (action.type) {
    ...
    default:
      return state;
}
```

Reply:
Hello @ShadyShawkat . Thank for the feedback